src/collections/resources/istio-ingress-gateway-in-kubernetes/index.mdx

```diff
@@ -0,0 +1,247 @@
+    What we do have to care about in the IngressGateway Deployment is SSL certificates. To be able to access the certificates inside the Gateway resources, make sure that you have mounted the certificates properly.
+</p>
+
+```yaml
+
+# Example represents volume mounts
+volumeMounts:
+- mountPath: /etc/istio/ingressgateway-certs
+  name: ingressgateway-certs
+  readOnly: true
+- mountPath: /etc/istio/ingressgateway-ca-certs
+  name: ingressgateway-ca-certs
+  readOnly: true
+
+# Example represents volumes
+volumes:
+- name: ingressgateway-certs
+  secret:
+    defaultMode: 420
+    optional: true
+    secretName: istio-ingressgateway-certs
+- name: ingressgateway-ca-certs
+  secret:
+    defaultMode: 420
+    optional: true
+    secretName: istio-ingressgateway-ca-certs
+```
+
+<h2>The Gateway</h2>
+
+<p>
+    The Gateway resources are used to configure the ports for Envoy. 
+    Since we have exposed three ports with the service, we need these ports to be handled by Envoy. We can do this by declaring one or more Gateways. 
+    In my example, I'm going to use a single Gateway, but it may be split into two or three.
+</p>
+
+```yaml
+
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: default-gateway
+  namespace: istio-system
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+
+  - hosts:
+    - '*'
+    port:
+      name: http
+      number: 80
+      protocol: HTTP
+
+  - hosts:
+    - '*'
+    port:
+      name: https
+      number: 443
+      protocol: HTTPS
+    tls:
+      mode: SIMPLE
+      privateKey: /etc/istio/ingressgateway-certs/tls.key
+      serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
+
+  - hosts: # For TCP routing this fields seems to be ignored, but it is matched
+    - '*'  # with the VirtualService, I use * since it will match anything.
+    port:
+      name: mysql
+      number: 3306
+      protocol: TCP
+```
+
+<h2>VirtualService</h2>
+<p>
+    The final interesting resource is the VirtualService, it works in concert with the Gateway to configure Envoy. 
+    If you only add a Gateway nothing will show up in the Envoy configuration, and the same is true if you only add a VirtualService.
+</p>
+<p>
+    A general configuration for an HTTP(s) service
+</p>
+
+```yaml
+
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: counter
+spec:
+  gateways:
+  - default-gateway.istio-system.svc.cluster.local
+  hosts:
+  - counter.lab.example.com
+  http:
+  - match:
+    - uri:
+      prefix: /
+    route:
+    - destination:
+        host: counter
+        port:
+          number: 80
+
+```
+<h2>Application Service and Deployment</h2>
+<p>
+    Our request have now reached the application service and deployment. These are just normal Kubernetes resources and I will assume that if you have read this far, you already know all about it. 😃
```

Comment:
@Chadha93 I missed reviewing this before 😅. So since this goes under `/resources`, we avoid using first person point of view while writing. Can you please go through this piece and make those changes in a separate PR? ( basically avoid mentioning "I will assume..." , "I have described above...", etc.)

Reply:
Oh, nice catch @debo19 Sure will update this